app/serializers/api/v2/sections/chapter_serializer.rb

```diff
@@ -8,7 +8,7 @@ class ChapterSerializer
 
         set_id :goods_nomenclature_sid
 
-        attributes :goods_nomenclature_sid, :goods_nomenclature_item_id, :headings_from, :headings_to, :description, :formatted_description
+        attributes :goods_nomenclature_sid, :goods_nomenclature_item_id, :headings_from, :headings_to, :description, :formatted_description, :validity_start_date, :validity_end_date
```

Comment:
this is a very looong line (we should avoid line > 120 chars long),
What about multiple rows? :

```
attributes :goods_nomenclature_sid,
           :goods_nomenclature_item_id,
           :headings_from,
           :headings_to,
           :description,
           :formatted_description,
           :validity_start_date,
           :validity_end_date
```